[Doc Improvement][InTune support dev preview][3597403] (#10298)
MicrosoftDocs/msteams-docs: msteams-platform/resources/schema/manifest-schema-dev-preview.md

|`PageId`| String | |  | Maps to the `EntityId` of the static tab. |
|`SubPageId`| String | |  | Maps to the `SubEntityId` of the static tab. |

## intuneInfo

**Optional** &ndash; Object

Properties related to app support for Microsoft Intune.

|Name| Type| Maximum size | Required | Description|
|---|---|---|---|---|

> |`supportedMobileAppManagementVersion`| String | 64 characters |  | Supported Microsoft Intune mobile app management (MAM) version that the app is compliant with. Expected value is a single version number, in the form of `integer.integer` (for example, `1.2`), that represents the maximum version of support to which this app attests. If no value is specified, the app does not attest to be mobile app management compliant. |

Reviewer: ```suggestion
|`supportedMobileAppManagementVersion`| String | 64 characters |  | Supported Microsoft Intune Mobile App Management (MAM) version. The value is a single version number in the `integer.integer` format, such as `1.2`, indicating the highest level of support the app confirms. If no value is provided, the app doesn't comply with mobile app management. |
```

Reviewer: Hi @erikadoyle,

What does "app is compliant with" here mean? If we remove it, does it change the meaning of the description?

If we are trying to say that "an app is compliant if it has a MAM version", I believe we don't need to mention it in the description in the manifest. Please correct me if my understanding is incorrect.

thanks,
Yash

This item is an array of `dashboardCard` elements of type `object`.

### dashboardCards.dashboardCard

Defines a single dashboard card and its properties.

|Name| Type| Maximum size | Required | Description|
|---|---|---|---|---|
|`id`| String | | ✔️ |  A unique identifier for this dashboard card. ID must be a GUID. |
|`displayName`| String | 255 characters | ✔️ | Display name of the card.|
|`description`| String | 255 characters | ✔️ | Description of the card.|
|`pickerGroupId`| String | | ✔️ | ID of the group in the card picker. ID must be a GUID.|
|`icon`| Object | | | Specifies icon for the card. |
|`contentSource`| Object | | ✔️ | Specifies the source of the card's content |
|`defaultSize`| String | | ✔️ | Rendering size for the dashboard card. Options: `medium` or `large`. |

### dashboardCards.dashboardCard.icon

Defines the icon properties of a given dashboard card.

|Name| Type| Maximum size | Required | Description|
|---|---|---|---|---|
|`iconUrl`| String | 2048 characters | | Location of the icon for the card, to be displayed in the toolbox and card bar. |

> |`officeUIFabricIconName`| String | 255 characters | | Office UI Fabric/Fluent UI icon friendly name for the card. This value will be used if *iconUrl* is not specified. |

Reviewer: ```suggestion
|`officeUIFabricIconName`| String | 255 characters | | Office UI Fabric or Fluent UI icon friendly name for the card. This value is used if *iconUrl* is not specified. |
```
